.travis.yml

```diff
@@ -2,19 +2,24 @@ esudo: false
 branches:
   only:
   - master
+  - develop
 language: node_js
 node_js:
 - '6'
 before_install:
 - 'pwd && ls -als'
+- curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 0.23.2
+- export PATH="$HOME/.yarn/bin:$PATH"
```

Comment:
I wonder why this is needed. Yarn is automatically available it seems:
https://github.com/edrlab/r2-streamer-js/blob/develop/.travis.yml

https://travis-ci.org/edrlab/r2-streamer-js